src/modules/configs/config.py

```diff
@@ -0,0 +1,48 @@
+import os
+from json import loads
+from typing import Union
+
+from jsonschema import validate, ValidationError
+
+from ingestation.modules.global_vars import APPLICATION_DIR
+
+
+def validate_json(conf: dict, schema_file) -> None:
+    validate(instance=conf, schema=getattr(schema_file, "schema"))
+
+
+class Conf:
+    """Class for parsing configuration files"""
+
+    def __init__(self, conf_path: Union[bool, str], conf_type: str, schema_file):
+        self.conf_type = conf_type
+        self.conf_path = self.get_conf_path(conf_path)
+        self.schema_file = schema_file
+
+    def parse_conf(self) -> Union[bool, dict]:
+        conf = self.load_conf()
+
+        try:
```

Comment:
I think this exc handling should be done directly in `validate_json()` - this way you don't have to repeat it in `MergedConf.create_final_conf()`.

Reply:
ok, tested that can get config type from schema_file name